pcsx2/R5900.h

```diff
@@ -260,6 +260,7 @@ alignas(16) extern fpuRegisters fpuRegs;
 alignas(16) extern tlbs tlb[48];
 
 extern u32 g_nextEventCycle;
+extern u32 g_lastEventCycle;
```

Comment:
Should this be serialized/saved to state? (given it's accessed in HwRead, which doesn't have cpuEventTest in its path).

Reply:
It could be, but it doesn't really matter since it will basically get updated pretty darn quickly and it's only used for the INTC hack. So it could be, I just couldn't justify bumping savestates just for that.